GoodDollar/GoodDAPP
# #1458 claim queue

# Description

see issue #1458 for description and cases

# How to test
- make sure claim queue env toggle is true
- make sure on server claim queue allowed is >0 to enable queue
- after "no spots" case is tested it is possible to add more spots by calling server api
/admin/queue with body: `{allow:number, password: gundbpassword}`
